src/components/taskbar/index.js

```diff
@@ -54,9 +54,9 @@ const TaskBar = ({ classes }) => (
       <span className={classes.subLogoText}>ALPHA</span>
     </View>
     <View className={classes.buttons}>
-      <LinkButton text="Swap" to="/" />
-      <LinkButton text="Refund" to="/refund" />
-      <LinkButton text="FAQ" to="/faq" />
+      <LinkButton text="Swap" onPress={() => goHome()} />
+      <LinkButton text="Refund" onPress={() => goRefund()} />
+      <LinkButton text="FAQ" onPress={() => window.alert('WIP')} />
       <LinkButton
         external
         text="Github"
```

Comment:
I think its better to link the boltz organisation and not only the frontend repo